src/shogun/kernel/Kernel.h

```diff
@@ -62,6 +62,7 @@ enum EKernelType
 	K_GAUSSIAN = 30,
 	K_GAUSSIANSHIFT = 32,
 	K_GAUSSIANMATCH = 33,
+	K_GAUSSIANFITC = 34,
```

Comment:
Make sure to document this properly. Having all these different Gaussian kernels might get confusing otherwise.

I would still prefer a *single* GaussianKernel which can enable ARD via a flag and which also containts all the stuff needed for FITC inference. Though it might be good so seperate due to seperation of concerns rule. So let's leave it this way.